ccf/tests/test_models.py

```diff
@@ -0,0 +1,30 @@
+import django
+import pytest
+from app.models import Jobs
+
+
+@pytest.mark.django_db()
+def test_create_job_entry():
+    job = Jobs(
+        finished=False, dockerfile="this is a dockerfile.", datastore_link="abc.com"
+    )
+
+    job.save()
+
+    assert len(Jobs.objects.all()) == 1
+
+
+@pytest.mark.django_db()
+def test_creating_job_without_dockerfile_raises_integrity_error():
+    job = Jobs(finished=False, datastore_link="abc.com")
```

Comment:
What happens if you drop the `finished` argument?

Reply:
Has a default, but good call, will test the default

Reply:
Addressed in https://github.com/Ross-Alexandra/SENG-371-Project-2/pull/43/commits/4bedbac1592ea65e07e8b93ec5f726d8d950774d